modules/Layout/Header/LanguagesDropdown/LanguagesDropdown.tsx

```diff
@@ -44,6 +45,27 @@ function LanguagesDropdown({ buttonClassName, navigationItemClassName, hasError
         );
     }, [currentLocale, languages]);
 
+    // Dynamically update the language attributes on `html` and `meta` tags when changing language.
```

Comment:
This is the same fix that was done for Privacy Portal: https://github.com/prezly/privacy-portal/commit/a11027cf1ebce8e9dedf00b09827d4e9753ad052

Reply:
Isn't there a full page refresh when you change culture via the dropdown? Because otherwise I don't see a point in dynamically changing it.

Reply:
We also don't support dynamic translations since we've dropped that in favor of server-side loading, right?

Reply:
> Isn't there a full page refresh when you change culture via the dropdown? Because otherwise I don't see a point in dynamically changing it.

No, the way Next works, after initial page load, all other subsequent page changes are handled on the client.
Translations are updated dynamically, since the server-side props are still requested on page change.

Basically, everything other than `_document` is re-rendering on page changes.

Reply:
> No, the way Next works, after initial page load, all other subsequent page changes are handled on the client.

I know that's how it works normally but in `LanguagePicker` we are forcing a refresh and I can see in the Network tab that it actually does a full refresh and also the `lang` attribute on `<html>` gets updated.

The language picker works dynamically in privacy portal due to the way it's coded but not in Bea.

Reply:
So I don't think we need this code at all.

Reply:
> The language picker works dynamically in privacy portal due to the way it's coded but not in Bea.

Oh, you're right!
So maybe I'd be better off doing the same for Privacy Portal 🤔 
I'll revert the commit then.